instrumentation/httpurlconnection/library/lint-baseline.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file? Is it needed?

Reply:
As our minCompileSDK is 21 in otel.android-app-conventions.gradle.kts and in HttpUrlReplacements.java we use some API 24 functions like getContentLengthLong(), we need to define a lint baseline file like this to prevent compile errors.

Reply:
can we use either`@SuppressLint("NewApi")` or androidx `RequiresApi` method annotations or even do a runtime check for the SDK version? I think they will make a lint pass and you don't need this extra file that is a bit uglier to read.
also, this file has to be regenerated when the AGP version changes or the line number of the suppressed issues changes as well, so more maintenance.

Reply:
That looks better! Updated to add suppress annotation. We do not need a sdk check or requiresApi annotation as it is a replacement function, will only be replaced for original API 24 calls.